packages/flutter/test/material/slider_test.dart

```diff
@@ -118,6 +118,95 @@ void main() {
     log.clear();
   });
 
+  testWidgets('Slider has a customizable active color',
+      (WidgetTester tester) async {
+    final Color customColor = const Color(0xFF4CD964);
+    final ThemeData theme = new ThemeData(platform: TargetPlatform.android);
+    Widget buildApp(Color activeColor) {
+      return new Material(
+        child: new Center(
+          child: new Theme(
+            data: theme,
+            child: new Slider(
+              value: 0.5,
+              activeColor: activeColor,
+              onChanged: (double newValue) {},
+            ),
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildApp(null));
+
+    final RenderBox sliderBox =
+        tester.firstRenderObject<RenderBox>(find.byType(Slider));
+
+    expect(sliderBox, paints..rect(color: theme.accentColor)..rect(color: theme.unselectedWidgetColor));
+    expect(sliderBox, paints..circle(color: theme.accentColor));
+    await tester.pumpWidget(buildApp(customColor));
+    expect(sliderBox, paints..rect(color: customColor)..rect(color: theme.unselectedWidgetColor));
+    expect(sliderBox, paints..circle(color: customColor));
```

Comment:
you probably want some negative tests here too, verifying that there isn't a circle of the custom color when the custom color isn't set and that there isn't one of the accent color when the custom color _is_ set.